src/gui/src/timingWidget.cpp

```diff
@@ -308,7 +306,10 @@ void TimingWidget::writeSettings(QSettings* settings)
 {
   settings->beginGroup(objectName());
 
-  settings->setValue("path_count", settings_->getPathCount());
+  // NOTE! Don't save path count, this can introduce problems when
+  // switching between large and small designs. Clicking Timing Report
+  // Update for a large number of paths will freeze the GUI, effectively
+  // for large designs.
```

Comment:
it would be sufficient to just avoid loading the value (but still save it).

Reply:
better?

Reply:
would not loading the settings file be sufficient instead?

Reply:
I still want saved settings, I just don't want the settings that can lock up(as in it is faster to kill the GUI and reload) the GUI for large designs.

Reply:
I mean something like this: https://github.com/The-OpenROAD-Project/OpenROAD/pull/5929

Reply:
Seems useful, but I don't understand the connection to this PR. In this PR, I'm trying to address a specific option that is causing the GUI to "lock up".

Reply:
no real connection, except it would also allow you to avoid loading the path_count, while preserving the functionality to save and load it until a solution to the speed is addressed. My personal preference would be to file an issue on the speed thing and see if it can be addressed before changing the default behavior for the GUI.

Reply:
I just assumed it would take a considerable amount of time to address so I skipped that step...

Wrong?

Reply:
@oharboe without a testcase and a filed issue, then yes, it will take a very long time to get addressed. Please open an issue so it can be addressed at the root of the issue.

Reply:
feature request added, this PR is "boarding up the broken window".

https://github.com/The-OpenROAD-Project/OpenROAD/issues/5931